src/options.cpp

```diff
@@ -1302,36 +1303,48 @@ std::vector<options_manager::id_and_option> options_manager::get_lang_options()
-        { "en", no_translation( R"(English)" ) },
-        { "ar", no_translation( R"(العربية)" )},
-        { "cs", no_translation( R"(Český Jazyk)" )},
-        { "da", no_translation( R"(Dansk)" )},
-        { "de", no_translation( R"(Deutsch)" ) },
-        { "el", no_translation( R"(Ελληνικά)" )},
-        { "es_AR", no_translation( R"(Español (Argentina))" ) },
-        { "es_ES", no_translation( R"(Español (España))" ) },
-        { "fr", no_translation( R"(Français)" ) },
-        { "hu", no_translation( R"(Magyar)" ) },
-        { "id", no_translation( R"(Bahasa Indonesia)" )},
-        { "is", no_translation( R"(Íslenska)" )},
-        { "it_IT", no_translation( R"(Italiano)" )},
-        { "ja", no_translation( R"(日本語)" ) },
-        { "ko", no_translation( R"(한국어)" ) },
-        { "nb", no_translation( R"(Norsk)" )},
-        { "nl", no_translation( R"(Nederlands)" )},
-        { "pl", no_translation( R"(Polski)" ) },
-        { "pt_BR", no_translation( R"(Português (Brasil))" )},
-        { "ru", no_translation( R"(Русский)" ) },
-        { "sr", no_translation( R"(Српски)" )},
-        { "tr", no_translation( R"(Türkçe)" )},
-        { "uk_UA", no_translation( R"(український)" )},
-        { "zh_CN", no_translation( R"(中文 (天朝))" ) },
-        { "zh_TW", no_translation( R"(中文 (台灣))" ) },
     };
 
+    constexpr std::array<std::pair<const char *, const char *>, 25> language_names = {{
+            // Note: language names are in their own language and are *not* translated at all.
+            // Note: Somewhere in Github PR was better link to msdn.microsoft.com with language names.
+            // http://en.wikipedia.org/wiki/List_of_language_names
+            { "en", R"(English)" },
+            { "ar", R"(العربية)" },
+            { "cs", R"(Český Jazyk)" },
+            { "da", R"(Dansk)" },
+            { "de", R"(Deutsch)" },
+            { "el", R"(Ελληνικά)" },
+            { "es_AR", R"(Español (Argentina))" },
+            { "es_ES", R"(Español (España))" },
+            { "fr", R"(Français)" },
+            { "hu", R"(Magyar)" },
+            { "id", R"(Bahasa Indonesia)" },
+            { "is", R"(Íslenska)" },
+            { "it_IT", R"(Italiano)" },
+            { "ja", R"(日本語)" },
+            { "ko", R"(한국어)" },
+            { "nb", R"(Norsk)" },
+            { "nl", R"(Nederlands)" },
+            { "pl", R"(Polski)" },
+            { "pt_BR", R"(Português (Brasil))" },
+            { "ru", R"(Русский)" },
+            { "sr", R"(Српски)" },
+            { "tr", R"(Türkçe)" },
+            { "uk_UA", R"(український)" },
+            { "zh_CN", R"(中文 (天朝))" },
+            { "zh_TW", R"(中文 (台灣))" },
+        }
+    };
+
+    for( const auto& [lang_id, lang_name] : language_names ) {
+        std::string name = lang_name;
+        if( const lang_stats *stats = lang_stats_for( lang_id ) ) {
+            name += string_format( _( " (%.1f%%)" ), stats->percent_translated() );
```

Comment:
Some color or better formatting would be nice here. This is just a quick example:
```suggestion
            name += string_format( colorize( _( " (%.1f%%)" ), c_dark_gray ), stats->percent_translated() );
```
<details>
<summary>screenshot</summary>

![Screenshot from 2023-04-08 06-30-18](https://user-images.githubusercontent.com/68240139/230701336-9504a809-c18e-418b-a8dc-d1cf7d14fbb2.png)

</details>

But this needs either a change in order of initialization (`init_colors()` before `get_options().init()`) or a manual color tag.

Reply:
Good idea.  I opted for a manual color tag since I'm too scared to rearrange initialization order.